o8g/Extractor/config.xml

```diff
@@ -0,0 +1,76 @@
+﻿<xml>
+  <game cardsUrl="https://api.fiveringsdb.com/cards"
+        packsUrl="https://api.fiveringsdb.com/packs"
+        gameGuid="bb0f02e7-2a6f-4ae3-84a2-c501b4176844"
+        packId="id"
+        cardNumber="position"
+        cardOctgnNumber="Card Number"
+        cardName="name"
+        cardPackId="pack_code"
+        cardId="octgn_id"
+        cardImage="image_url"
+        />
+  <images>
+    <image name="FiveringsDB" url="{0}" />
+  </images>
+  <properties>
+    <property octgn_name="Clan" type="property" value="clan" capitalize="True" />
+    <property octgn_name="Text" type="property" value="text" isRich="True"/>
+    <property octgn_name="Type" type="property" value="type" capitalize="True" />
+    <property octgn_name="Traits" type="property" value="traits" capitalize="True" delimiter=". " />
+    <property octgn_name="Ring" type="property" value="element" capitalize="True" />
+    <property octgn_name="Card Number" type="property" value="position" />
+    <property octgn_name="Cost" type="property" value="cost" />
+    <property octgn_name="Military Skill" type="property" value="military" />
+    <property octgn_name="Military Skill" type="property" value="military_bonus" />
+    <property octgn_name="Political Skill" type="property" value="political" />
+    <property octgn_name="Political Skill" type="property" value="political_skill" />
+    <property octgn_name="Bonus Strength" type="property" value="strength" />
```

Comment:
The plugin was using a single field for these and expected them all to be prefixed with '+', is there an easy way to automatically add a '+' prefix to one of these since the raw data doesn't have it

Reply:
Oops, this isn't correct. Provinces don't have the '+' and aren't expected to. Nothing to see here, everything's as it's supposed to be.